src/connectors/autocomplete/connectAutocomplete.ts

```diff
@@ -0,0 +1,171 @@
+import {
+  AlgoliaSearchHelper as Helper,
+  SearchResults,
+} from 'algoliasearch-helper';
+import escapeHits, { TAG_PLACEHOLDER } from '../../lib/escape-highlight';
+import {
+  checkRendering,
+  createDocumentationMessageGenerator,
+  noop,
+  warning,
+} from '../../lib/utils';
+import {
+  RendererOptions,
+  Renderer,
+  WidgetFactory,
+  Hits,
+  InstantSearch,
+  Unmounter,
+} from '../../types';
+
+const withUsage = createDocumentationMessageGenerator({
+  name: 'autocomplete',
+  connector: true,
+});
+
+interface AutocompleteIndex {
+  index: string;
+  hits: Hits;
+  results: SearchResults | undefined;
```

Comment:
For which cases the value is `undefined`?

Reply:
None, it's mistyped.